### DISCUSSION TITLE
Python script for extracting images from output

### DISCUSSION THREAD

**Discussion by rnpratoori** [idaholab/moose#28746] [Q&A Tools] [answered]
I have run ~500 simulations and have the corresponding output files. I want to extract the images of the last timesteps of each of the files. I know how to do it with `*vtu` format using Python. I am not familiar with the `MultiBlock` type of writing output with `Exodus`. Can anyone tell me how to handle this data?

I am not sure if this is the right place to ask this question, but it would be great if someone could point me to a proper source on this.

**Comment by GiudGiud** [comment, marked as answer]
I think whichever python package you are working with for extracting the data from vtu files is a better place to start from.

Note that you can use VTK output in moose, exodus is not the only option
https://mooseframework.inl.gov/source/outputs/VTKOutput.html
